staging/src/k8s.io/client-go/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Sorry, client-go does not accept changes via pull requests at this time. Please
```

Comment:
Should we add this file everywhere?

Reply:
good idea, I'll take another pass at this